compair/static/modules/comment/comment-form-partial.html

```diff
@@ -28,20 +28,17 @@ <h2 ng-if="answerComment"><i class="fa fa-user"></i> {{parent.user.displayname}}
         </div>
     </fieldset>
 
-    <div class="row" ng-show="comment.draft">
-        <div class="col-md-3 col-md-offset-3 col-xs-6">
-            <input type="submit" class="btn btn-default btn-lg center-block" value="Save Draft"
-                ng-click="comment.draft = true"
-                ng-disabled="submitted" />
-        </div>
-        <div class="col-md-3 col-xs-6">
-            <input type="submit" class="btn btn-success btn-lg center-block" value="Save"
-                ng-click="comment.draft = false"
-                ng-disabled="commentForm.$invalid || submitted" />
-        </div>
+    <div class="form-group sub-step text-center" ng-show="comment.draft">
+        <input type="submit" class="btn btn-default btn-lg" value="Save Draft"
+            ng-click="comment.draft = true"
+            ng-disabled="submitted" />
+        &nbsp;&nbsp;
+        <input type="submit" class="btn btn-success btn-lg" value="Submit"
+            ng-click="comment.draft = false"
+            ng-disabled="commentForm.$invalid || submitted" />
     </div>
     <div ng-show="!comment.draft">
-        <input type="submit" class="btn btn-success btn-lg center-block" value="Save"
+        <input type="submit" class="btn btn-success btn-lg center-block" value="Submit"
```

Comment:
I believe we're leaving these as "Save"? So first you have options to "Save Draft" or "Submit", then you have the option only to "Save" (since it's already submitted at that point).

Reply:
I think they may still not have submitted it. The only condition for showing that is if there is no "Draft" self-evaluation.